dll/win32/shell32/CQueryAssociations.cpp

```diff
@@ -662,7 +666,7 @@ HRESULT CQueryAssociations::GetExecutable(LPCWSTR pszExtra, LPWSTR path, DWORD p
     WCHAR *pszEnd;
 
     HRESULT hr = this->GetCommand(pszExtra, &pszCommand);
-    if (FAILED(hr))
+    if (FAILED_UNEXPECTEDLY(hr))
```

Comment:
```suggestion
    if (FAILED(hr))
```

Reply:
However the FAILED_UNEXPECTEDLY themselves are fine I suppose. I was talking about the explicit extra ERRs that you added.

Reply:
Reverted.